packages/hoppscotch-common/src/platform/index.ts

```diff
@@ -9,10 +9,12 @@ import { AnalyticsPlatformDef } from "./analytics"
 import { InterceptorsPlatformDef } from "./interceptors"
 import { HoppModule } from "~/modules"
 import { InspectorsPlatformDef } from "./inspectors"
+import { Service } from "dioc"
 
 export type PlatformDef = {
   ui?: UIPlatformDef
   addedHoppModules?: HoppModule[]
+  addedServices?: Array<typeof Service<unknown>>
```

Comment:
Update this to fix the type issue in dioc module

```suggestion
  addedServices?: Array<typeof Service<unknown> & { ID: string }>
```